sdk/communication/azure-communication-jobrouter/azure/communication/jobrouter/_router_administration_client.py

```diff
@@ -0,0 +1,1422 @@
+        The credentials with which to authenticate
+    """
+
+    def __init__(
+            self,
+            endpoint: str,
+            credential: CommunicationTokenCredential,
+            **kwargs: Any
+    ) -> "None":
+        # type: (...) -> None
+        if not credential:
+            raise ValueError("credential can not be None")
+
+        try:
+            if not endpoint.lower().startswith('http'):
+                endpoint = "https://" + endpoint
+        except AttributeError:
+            raise ValueError("Host URL must be a string")
+
+        parsed_url = urlparse(endpoint.rstrip('/'))
+        if not parsed_url.netloc:
+            raise ValueError("Invalid URL: {}".format(endpoint))
+
+        self._endpoint = endpoint
+        self._authentication_policy = get_authentication_policy(endpoint, credential)
+        self._client = AzureCommunicationJobRouterService(
+            self._endpoint,
+            authentication_policy = self._authentication_policy,
+            sdk_moniker = SDK_MONIKER,
+            **kwargs)
+
+    @classmethod
+    def from_connection_string(
+            cls,
+            conn_str: str,
+            **kwargs: Any
+    ) -> "RouterAdministrationClient":
+        # type: (...) -> RouterAdministrationClient
+        """Create RouterClient from a Connection String.
+
+        :param str conn_str:
+            A connection string to an Azure Communication Service resource.
+        :return: Instance of RouterAdministrationClient.
+        :rtype: ~azure.communication.jobrouter.RouterAdministrationClient
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/sample_authentication.py
+                :start-after: [START admin_auth_from_connection_string]
+                :end-before: [END admin_auth_from_connection_string]
+                :language: python
+                :dedent: 8
+                :caption: Authenticating a RouterAdministrationClient from a connection_string
+        """
+        endpoint, access_key = parse_connection_str(conn_str)
+
+        return cls(endpoint, access_key, **kwargs)
+
+    # region ExceptionPolicy
+
+    @overload
+    def create_exception_policy(
+            self,
+            exception_policy_id: str,
+            exception_policy: ExceptionPolicy,
+            **kwargs: Any
+    ) -> ExceptionPolicy:
+        """ Create a new exception policy.
+
+        :param str exception_policy_id: Id of the exception policy.
+
+        :param exception_policy: An instance of exception policy. This is a positional-only parameter.
+          Please provide either this or individual keyword parameters.
+        :type exception_policy: ~azure.communication.jobrouter.ExceptionPolicy
+
+        :return: ExceptionPolicy
+        :rtype: ~azure.communication.jobrouter.ExceptionPolicy
+        :raises: ~azure.core.exceptions.HttpResponseError, ValueError
+        """
+
+    @overload
+    def create_exception_policy(
```

Comment:
Please remove the key-word overloads from the `create_` operations.

Reply:
All the `overload` of `create_` ops have been removed

Reply:
Then this should take an `ExceptionPolicy` directly as input :)